_posts/2020-04-05-error-handling.md

```diff
@@ -20,7 +20,7 @@ all wrong :-).
 
 ## Language syntax
 
-I know this is somewhat contentious topic. But I'm a strong opponent of adding
+I know this is a somewhat contentious topic. But I'm a strong opponent of adding
```

Comment:
This one sounds weird. I believe `some` (and `somewhat`) can act instead of `a`.

Reply:
I think that the word `somewhat`, in this context, doesn't have an impact on whether or not the `a` should be present. Please take into consideration that the following sentence:
```
I know this is a contentious topic.
```
The `a` here only refers to the `topic`.
Anyways, perhaps the best thing I can do here is give an [example](https://en.wikipedia.org/wiki/A_Somewhat_Gentle_Man "A Somewhat Gentle Man") :)

Reply:
I'd say that names of movies and literature doesn't really count as a reference on what is good use of English.

Seems like this source mostly agrees with me that there's no `a` before somewhat: https://www.merriam-webster.com/dictionary/somewhat.